requirements.dev.txt

```diff
@@ -1,10 +1,10 @@
 flake8
 # Lock mypy to the same version used in downstream build environments
-mypy==0.910
+mypy==0.931
```

Comment:
can this be pinned to "this or higher" ?

Reply:
Sure. I meant change from pinning to exact version to "same major and any minor tihs or higher"